ideascube/monitoring/forms.py

```diff
@@ -73,16 +74,17 @@ class Meta:
 
 class InventorySpecimenForm(forms.ModelForm):
 
-    specimen = forms.CharField(widget=forms.TextInput(attrs={
-                               'placeholder': _('Enter a bar code')}))
+    specimen = forms.CharField(label=_('specimen'),
```

Comment:
without `label=` for consistency with [previous usage](https://github.com/ideascube/ideascube/pull/530/files#diff-2f239666916c6ec0ead7cc06d8bee7efR91)